README.markdown

```diff
@@ -169,6 +169,13 @@ Synopsis
     ngx.say("AES 128 CBC (WITH IV) Encrypted HEX: ", str.to_hex(encrypted))
     ngx.say("AES 128 CBC (WITH IV) Decrypted: ",
         aes_128_cbc_with_iv:decrypt(encrypted))
+
+    local b64 = require "resty.base64"
+    ngx.say("base64 encoded: ", b64.encode_base64("foo"))
```

Comment:
Is this duplicate with `ngx_lua`'s `ngx.encode_base64()` and `ngx.decode_base64()` API functions?

https://github.com/openresty/lua-nginx-module/#ngxencode_base64

https://github.com/openresty/lua-nginx-module/#ngxdecode_base64